### DISCUSSION TITLE
king-of-mashing/

### DISCUSSION THREAD

**Discussion by giscus** [zmike/zmike.github.io#27] [Announcements]
# king-of-mashing/

Overdue

https://www.supergoodcode.com/king-of-mashing/

**Comment by EliasOfWaffle** [comment]
Sorry, i have a doubt, is possible to Zink use Mesh Shader Vulkan extension to implementing Geometry shaders, Tesselation Shader and if can be possible Computer shaders too?

**Reply by zmike** [reply]
Maybe
